docs: revenue share description optimisation for copyright-verfication mudule (#687)
taptap/tds-docs: i18n/en/docusaurus-plugin-content-docs/current/sdk/copyright-verification/features.mdx

At this moment, if you haven’t enabled Copyright Verification on the Developer Center, you won’t be able to access the Price Settings.
This means that even you only need to launch a paid game without using the Copyright Verification SDK in your game, you still need to enable the Copyright Verification feature on the Developer Center.
Once your application to enable the feature has been approved, you will be able to access the Price Settings.

> ## TapTap 

Reviewer: ```suggestion
## How much share does TapTap take?
```

> TapTap do not share in the revenue

Reviewer: ```suggestion
TapTap does not take a share of the revenue.
```

> However, there will be a 5% payment processing fee in the Mainland China region, and overseas will be determined based on the actual communication (processing fee and tax cost of third-party payment channels).

Reviewer: ```suggestion
However, there will be a 5% payment processing fee in the Mainland China region. For other countries and regions, the fee will be determined upon communication (the fee includes third-party payment channel processing fees and tax costs).
```
